hector_math/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.5.1)
 project(hector_math VERSION 1.0.0)
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Why the switch to 14?

Reply:
The master branch is also on 14, and the conditional types for pointer and reference look a little nicer with 14.